pkg/deploy/preload.go

```diff
@@ -54,6 +54,16 @@ func preloadCaches(ctx context.Context, projects []project.Project, clientSet *c
 	wg.Wait()
 }
 
+func clearCaches(clientSet *client.ClientSet) {
```

Comment:
I was considering if this should be moved to ClientSet, but considering how preload currently works, I think it makes more sense here.